active_directory_ldap/plugin.spec.yaml

```
@@ -261,6 +261,34 @@ actions:
         description: Operation status
         type: boolean
         required: false
+  modify_object:
+    title: Modify Object
+    description: Modify an Active Directory objects attributes
```

```suggestion
    description: Modify the attributes for an Active Directory object
```
This reads a little better IMO